alacritty.yml

```diff
@@ -532,7 +532,7 @@
 # be mapped to the `ReceiveChar` action. Alternatively, you can use `None` for
 # a no-op if you do not wish to receive input characters for that binding.
 #key_bindings:
-  # (Windows and Linux only)
+  # (Windows, Linux and BSD only)
```

Comment:
nit: Oxford comma, please

`(Windows, Linux and BSD only)` -> `(Windows, Linux**,** and BSD only)`